lib/app-layout/index.js

```diff
@@ -19,6 +22,7 @@ const NoteEditor = React.lazy(() =>
 );
 
 export const AppLayout = ({
+  filter = appState.filter,
```

Comment:
I'm kind of surprised that this works - it may be a by-product of our funky Redux setup. This is already being passed-in from `connect` and if we need a default it would probably be best to set it to something like `null` or an empty string rather than trying to grab `appState.filter`

Reply:
Apparently I don't need to set it, but I do need to pass it in ¯\_(ツ)_/¯